spec/README.mediawiki

```diff
@@ -1215,15 +1310,33 @@ for verification and relay to the counterparty.
 |-
 | matchid    || 8  || the ID assigned to this match
 |-
-| swap stage || 1  || 0 for initialization transaction, 1 for redemption
+| tx hash    || 32  || the transaction hash
+|-
+| vout       || 4 || the output index
+|-
+| timestamp  || int    || UTC timestamp. order broadcast time
+|-
+| sig        || string || DEX's signature of the serialized notification. serialization described below
+|}
+
+'''Redemption notification serialization'''
+
+{|
+! field      !! size (bytes)  !! description
+|-
+| orderid    || 16 || the order ID
+|-
+| matchid    || 8  || the ID assigned to this match
```

Comment:
I don't think we've defined how the match ID is computed.  This could be a monotonically increasing integer that the server tracks, or it could be a unique id computed from a hash or similar.  I don't recall any discussion around this.

Reply:
Kinda thinking just a counter. Do we need to say in the spec?

Reply:
Other than knowing the number of bytes required, I suppose it's not necessary to specify other than to make it clear it's a _unique_ ID, and that's pretty obvious.  The only reason I brought it up was because of the bytes.  64-bit integer is more than adequate for a counter, but it's something to track.  If there's a clever way to compute another kind of unique id without knowing order counter, that would be nice.